### PR TITLE
feat: Add `format_directories` option

### PR DESCRIPTION
**Pull request by pablojimpas**
## Description

Analogous to the `analyze_directories` option that already exists. Allows the user to configure for which directories wants to check formatting. Useful for excluding generated code, for example.

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Review by renancaraujo — APPROVED** [review]
This is very good 
Very good

File: README.md [hunk 1]
```diff
@@ -91,6 +91,12 @@ The Dart package workflow consists of the following steps:
 
 **Default** `"lib test"`
 
+#### `format_directories`
+
+**Optional** A space separated list of folders that should be formatted.
```
Comment (alestiago) [inline README.md:96]:
```suggestion
**Optional** A space separated list of directories that should be formatted.
```

Comment (alestiago) [inline README.md:96]:
```suggestion
**Optional** A space separated string of directories that should be formatted.
```

**Comment by renancaraujo** [comment]
Close and reopening to kick dead actions

**Comment by pablojimpas** [comment]
Is this ready to be merged? CI already passed @renancaraujo

**Comment by renancaraujo** [comment]
I was waiting for the 1.13 release, merging now

**Pull request merged**